src/components/MultiColumnMeasures.tsx

```diff
@@ -1,81 +1,85 @@
 import AddIcon from '@mui/icons-material/Add';
-import CancelIcon from '@mui/icons-material/Cancel';
-import { Fab, Card, CardContent, CardHeader, Typography, Tabs, Tab } from '@mui/material';
-import { useTheme } from '@mui/material/styles';
-import { useState } from 'react';
-import {
-  useMultiColumnMeasuresState,
-  useMultiColumnMeasuresData,
-  useActiveVariableData,
-} from '../hooks';
-import useDataStore from '../stores/data';
+import { Fab } from '@mui/material';
+import { SyntheticEvent } from 'react';
+import { useMultiColumnMeasureCardData } from '../hooks/useMultiColumnMeasureCardData';
+import { useMultiColumnMeasureDraftCard } from '../hooks/useMultiColumnMeasureDraftCard';
+import { useMultiColumnMeasureTabs } from '../hooks/useMultiColumnMeasureTabs';
+import { usePersistedMultiColumnCards } from '../hooks/usePersistedMultiColumnCards';
+import { useColumns, useFreshDataActions } from '../stores/FreshNewStore';
 import { MultiColumnMeasuresInstructions } from '../utils/instructions';
-import { MultiColumnMeasuresTerm } from '../utils/internal_types';
-import { getColumnsAssignedText, createMappedColumnHeaders } from '../utils/util';
 import Instruction from './Instruction';
+import MultiColumnMeasureTabs from './MultiColumnMeasureTabs';
 import MultiColumnMeasuresCard from './MultiColumnMeasuresCard';
+import MultiColumnMeasuresColumnsSidebar from './MultiColumnMeasuresColumnsSidebar';
 
 function MultiColumnMeasures() {
-  const theme = useTheme();
-  const [activeTab, setActiveTab] = useState(0);
-  const updateColumnStandardizedVariable = useDataStore(
-    (state) => state.updateColumnStandardizedVariable
+  const {
+    variables: multiColumnVariables,
+    activeTab,
+    activeVariable,
+    setActiveTab,
+  } = useMultiColumnMeasureTabs();
+  const activeVariableId = activeVariable?.id || '';
+  const persistedCards = usePersistedMultiColumnCards(activeVariableId);
+  const { draftMeasureCards, createDraft, updateDraftTerm, removeDraft, hasDraft } =
+    useMultiColumnMeasureDraftCard(activeVariableId);
+  const combinedCards = [...persistedCards, ...draftMeasureCards];
```

Comment:
We want the draft to show up at the end, does this spreading operation ensure the correct ordering?